client/pages/projects/index.jsx

```diff
@@ -1,3 +1,125 @@
-export default function ProjectsPage() {
-  return <>Temporary Page</>;
+import React, { Component } from "react";
+import { useEffect, useState } from "react";
+import { useRouter } from "next/router";
+import { Head } from "../../components";
+import PropTypes from "prop-types";
+import Select from "react-select";
+
+import "../../public/styles/home.scss";
+//import "../../public/styles/project.scss";
+import countryData from "../../utils/countries";
+import UNGoalData from "../../utils/goals";
+import groupSizeData from "../../utils/groups";
+import levelData from "../../utils/levels";
+import mockData from "../../utils/mockData";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faBell } from "@fortawesome/free-solid-svg-icons";
+import {
+  Button,
+  Card,
+  CardBody,
+  CardGroup,
+  CardImg,
+  CardText,
+  Col,
+  Container,
+  Input,
+  Row
+} from "reactstrap";
+
+const populateProjects = () => {
+  const numProjects = 21;
+
+  const projectsToReturn = new Array(numProjects).slice(numProjects);
+
+  for (let i = 1; i <= numProjects; i++) {
+    const projObject = {
```

Comment:
This is fine for now, but we'll work on pulling the data from the backend (since thats set up now) instead of using mockData.js. We'll see if we can get that done before MVP showcase starts